job_templates/sag_pt_model_learner/config_fed_client.json

```diff
@@ -0,0 +1,31 @@
+{
+    "format_version": 2,
+    "executors": [
+        {
+            "tasks": [
+                "train",
+                "submit_model",
+                "validate"
+            ],
+            "executor": {
+                "id": "Executor",
+                "path": "nvflare.app_common.executors.model_learner_executor.ModelLearnerExecutor",
+                "args": {
+                    "learner_id": "cifar10-learner"
+                }
+            }
+        }
+    ],
+    "task_result_filters": [],
+    "task_data_filters": [],
+    "components": [
+        {
+            "id": "cifar10-learner",
+            "path": "fl.train_model_learner.CIFAR10ModelLearner",
```

Comment:
Suggest you use file format ".conf" (please refer to "sag_pt")
And add some comments here to explain that users might need to change this if they are using different "Learner"